protocol/x/prices/keeper/market_param.go

```diff
@@ -50,6 +50,15 @@ func (k Keeper) ModifyMarketParam(
 		}
 	}
 
+	// Validate that modified market param has a corresponding ticker in MarketMap
+	cp, err := slinky.MarketPairToCurrencyPair(updatedMarketParam.Pair)
+	if err != nil {
```

Comment:
nit: Is this error case already covered by a separate test / un-reachable due to validation done before hand on the pair?

Reply:
Looks like the only caller of `ModifyMarketParam` is `UpdateMarketParam` in the `prices` keeper. That function doesn't perform this check, and none of its callers do either AFAICT (thought it starts getting tough to trace through the RPC calls). 

I also ran a quick search on `MarketPairToCurrencyPair` func and didn't see references to it in related / surrounding code paths. 

Just noticed we actually check the same condition on line 73 of this file: https://github.com/dydxprotocol/v4-chain/pull/2151/files#diff-b5fe4e18251360d63261212057a8fd1fb2520da6bbc27eae56c709c08a6bf0abR73

Which suggests this check is necessary but probably can be de-duped with the existing one